about.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Flawless Shop</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+<body>
+  <section class="about">
+    <nav class="nav">
+      <div class="nav-icons">
+        <ul>
+         <li><a href=""><i class="fa fa-twitter"></i></a></li> 
+         <li><a href=""><i class="fa fa-instagram"></i></a></li> 
+         <li><a href="">French</a></li>
+         <li><a href="">My page</a></li>
+         <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-bar">
+        <a href="home.html"><img class="click" src="images/logo.png" alt="logo"></a>
+        <div class="nav-links">
+          <a class="LINK" href="home.html">Home</a>
+          <a class="LINK"href="#Awards">Awards</a>
+          <a href="" class="flawless">FLAWLESS</a>
+        </div>
+      </div>
+      <div class="navicons">
+        <a href="home.html"><i class="fa fa-home"></i></a>
+        <i class="fa fa-angle-right"></i>
+        <p class="id">purchase</p>
+        <i class="fa fa-angle-right"></i>
+        <p class="id">My orders</p>
+      </div>
+      <i class="fa fa-bars fa-2x" id="bars"></i>
+    </nav>
+      <div class="nav-menu">
+        <i class="fa fa-times fa-2x" id="closingIcon"></i>
+        <ul>
+          <li class="mobileLinks"><a href="home.html">Home</a></li>
+          <li class="mobileLinks"><a href="#products">Products</a></li>
+          <li class="mobileLinks"><a href="#Awards">Awards</a></li>
+        </ul>
+      </div>
+    <div class="main2">
+      <q>Hi there!</q>
+      <h1 class="slogan">Give your skin a little love with Flawless!</h1>
+      <p class="description2">
+        Looking young and beautiful never comes without using the best cosmetics.
+        Here we have compiled the best skincare products by the best brands. If it's your first time joining us, we hope that you like what you're seeing.
+      </p>
+      <p class="contact-us">
+        You can contact us per Email for any further questions about Flawless. <br>
+        <a class="mail" href="">flawlessdc@gmail.com</a>
+      </p>
+    </div>
+  </section>
+  <div class="our-logo">
+    <h2 class="art">Flawless Logo</h2>
+    <hr class="title-underline">
+    <div class="logo-description">
+      <p class="paragraphe">The logo of Flawless was decided through the logo competition from 31/03 to 26/04 2012</p>
+      <a href=""><img class="logoo" src="images/logo.png" alt="logo"></a>
```

Comment:
- [ ] I have noticed that the facebook logo overlaps into the items below at screen size of around 740px. This makes the application less user-friendly. Kindly fix this to improve the User Interface ☺️.

Your design 😎 
![image](https://github.com/mariegrace31/capstone-project/assets/99394374/27efa75d-fd5e-4d05-b46e-730e48afcd58)